Enigmatry.Entry.Blueprint.Api/Program.cs

```diff
@@ -1,38 +1,28 @@
-﻿using Autofac;
-using Autofac.Extensions.DependencyInjection;
-using Enigmatry.Entry.Blueprint.Api;
+﻿using Enigmatry.Entry.Blueprint.Api;
 using Enigmatry.Entry.Blueprint.Infrastructure.Api.Init;
 using Enigmatry.Entry.Blueprint.Infrastructure.Init;
 using Serilog;
 
+var bootstrapConfiguration = ConfigurationHelper.CreateConfiguration(args);
 Log.Logger = new LoggerConfiguration()
-    .WriteTo.Console()
+    .AppConfigureSerilog(bootstrapConfiguration)
```

Comment:
I think this is good, I prefer to have the BootstrapLogger writting the the file log instead of Console.

Reply:
I agree